app/hub/Views/SideNavigationView/SideNavigationViewContainer.jsx

```diff
@@ -47,7 +46,7 @@ class SideNavigationViewContainer extends Component {
 	 */
 	render() {
 		const { user, location } = this.props;
-		const disableRegEx = /^(\/setup)|(\/tutorial)/;
+		const disableRegEx = /^(\/setup(?!\/4))|(\/tutorial(?!\/6))/;
```

Comment:
put a `$` character after 4 and 6 to make sure that it is matching to the last character of the route